frontend/src/components/Header.tsx

```diff
@@ -161,21 +159,23 @@ const Header = ({ logoPath, locale }: Props) => {
         }}
       />
       <GovBanner language={language} />
-      <USWDSHeader basic={true}>
+      <USWDSHeader
+        basic={true}
+        className="desktop:position-sticky top-0 desktop:z-500 bg-white border-bottom-2px border-primary-vivid"
```

Comment:
Added the bottom border here to match the Figma. Interestingly, this visual border comes from various places at different breakpoints. Since we always want a border, I added it in one place and explicitly remove the border on `#main-content` and `.usa-navbar`.